addons/common/stringtable.xml

```diff
@@ -68,105 +69,139 @@
+            <German>Stark</German>
         </Key>
         <Key ID="STR_ZEN_Common_Extreme">
             <English>Extreme</English>
+            <German>Extrem</German>
         </Key>
         <Key ID="STR_ZEN_Common_All">
             <English>All</English>
+            <German>Alle</German>
         </Key>
         <Key ID="STR_ZEN_Common_Side">
             <English>Side</English>
+            <German>Seite</German>
         </Key>
         <Key ID="STR_ZEN_Common_Faction">
             <English>Faction</English>
+            <German>Fraktion</German>
         </Key>
         <Key ID="STR_ZEN_Common_Aircraft">
             <English>Aircraft</English>
+            <German>Fluggerät</German>
         </Key>
         <Key ID="STR_ZEN_Common_Filter">
             <English>Filter</English>
+            <German>Filter</German>
         </Key>
         <Key ID="STR_ZEN_Common_Units">
             <English>Units</English>
+            <German>Einheiten</German>
         </Key>
         <Key ID="STR_ZEN_Common_Vehicles">
             <English>Vehicles</English>
+            <German>Fahrzeuge</German>
         </Key>
         <Key ID="STR_ZEN_Common_Static">
             <English>Static</English>
+            <German>Statische Waffen</German>
         </Key>
         <Key ID="STR_ZEN_Common_Nearest">
             <English>Nearest</English>
+            <German>Am nächsten</German>
         </Key>
         <Key ID="STR_ZEN_Common_Speed">
             <English>Speed</English>
+            <German>Geschwindigkeit</German>
         </Key>
         <Key ID="STR_ZEN_Common_Range">
             <English>Range</English>
+            <German>Reichweite</German>
         </Key>
         <Key ID="STR_ZEN_Common_Radius">
             <English>Radius</English>
+            <German>Radius</German>
         </Key>
         <Key ID="STR_ZEN_Common_Height">
             <English>Height</English>
+            <German>Höhe</German>
         </Key>
         <Key ID="STR_ZEN_Common_Height_Units">
             <English>Height (m)</English>
+            <German>Höhe (m)</German>
         </Key>
         <Key ID="STR_ZEN_Common_AutoAddObjects">
             <English>Add Objects to Curators</English>
+            <German>Objekte zum Kurator hinzufügen</German>
         </Key>
         <Key ID="STR_ZEN_Common_AutoAddObjects_Description">
             <English>Automatically adds spawned objects to all curators</English>
+            <German>Fügt automatisch gespawnte Objekte zu allen Kuratoren hinzu.</German>
         </Key>
         <Key ID="STR_ZEN_Common_DisableGearAnim">
             <English>Disable Gear Animation</English>
+            <German>Zahnrad Animation deaktivieren</German>
         </Key>
         <Key ID="STR_ZEN_Common_DisableGearAnim_Description">
             <English>Disables the "Gear" animation from playing when entering Zeus</English>
+            <German>Deaktiviert die "Zahnrat" Animation wenn man in dem Zeusmodus geht</German>
```

Comment:
lol, he means the animation, when you open your inventory.

Reply:
Didn't know what meant with this, I now heard means the animation the unit does while in zeus mode. For some reason my character never does it so i was kind of confused.

But yeah good to know, gonna change this up.

Reply:
Would be "Zahnrad" but it's actually "Ausrüstung"
It's the same as when you open inventory and lean forward and grapple around on your hips.